src/worker/shutdown.sh

```diff
@@ -2,33 +2,50 @@
 
 arch="nvidia"
 
+remove_docker=$1
+
 if [ -e '/dev/nvidiactl' ]; then
     arch="nvidia"
 elif [ -e '/dev/kfd' ];then
     arch="amd"
 else
-	arch='cpu'
+    arch='cpu'
 fi
 
 kill_exporters() {
     pkill -f "${1}_exporter.py*"
-	pkill -f "net_exporter.py*"
-	pkill -f "node_exporter.py*"
+    pkill -f "net_exporter.py*"
+    pkill -f "node_exporter.py*"
+}
+
+kill_docker() {
+	if [[ $(docker ps -a | grep genevamdmagent) ]]; then
+		echo "Stopping Geneva Metrics Extension(MA) container"
+		docker stop genevamdmagent
+		docker rm genevamdmagent
```

Comment:
do we need to remove docker image as well during shutdown?

Reply:
If we keep the image on the VM it will save time during install/start step if they desire to use the container again. This way we don't have to pull down the same image again.